test/e2e/cloud/framework.go

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
The similar type of disruptive tests live under /test/e2e/lifecycle/ like reboot, restart, lease.
So how about putting this also under the path?